assets/xml/overlays/ovl_fbdemo_wipe3.xml

```diff
@@ -0,0 +1,13 @@
+<Root>
+    <File Name="ovl_fbdemo_wipe3" BaseAddress="0x80AC6740" RangeStart="0x520" RangeEnd="0x1C10">
+        <Texture Name="fbdemo_tex_000520" Format="i8" Width="16" Height="64" Offset="0x520"/>
+        <Texture Name="fbdemo_tex_000920" Format="i8" Width="16" Height="64" Offset="0x920"/>
+        <Texture Name="fbdemo_tex_000D20" Format="i8" Width="16" Height="64" Offset="0xD20"/>
+        <Texture Name="fbdemo_tex_001120" Format="i8" Width="16" Height="64" Offset="0x1120"/>
+        <Texture Name="fbdemo_tex_001520" Format="i8" Width="16" Height="64" Offset="0x1520"/>
```

Comment:
Could you add `OutName`'s to these textures? Looks like unnamed textures just use `tex_000520` as their outname